### PR TITLE
Fix/shiny and jupyter

### LINKED ISSUE #46: ds-rshiny and ds-jupyter-notebeook: ready for ODS2
**Is your feature request related to a problem? Please describe.**
Both `ds-rshiny` and `ds-jupyter-notebook` requires getting latest changes and some refactor on the provisioning side.

**Describe the solution you'd like**
To get the changes from https://github.com/opendevstack/ods-project-quickstarters/issues/424.
And, to get independent from ods-configuration (enhance the provisioning)

**Additional context**
If we do not apply this changes the provisioning will be failing!

### PR DESCRIPTION
**Pull request by gerardcl**
closes  #46

### CODE REVIEW

File: common/scripts/create-rshiny.sh [hunk 1]
```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
```
Comment (michaelsauter) [inline common/scripts/create-rshiny.sh]: @gerardcl Could you move that script out of `common`? I would put this inside the rshiny folder.

Comment (gerardcl) [reply common/scripts/create-rshiny.sh]: @michaelsauter shall I move out the Tailor-related files from common too?

Comment (michaelsauter) [reply common/scripts/create-rshiny.sh]: `common` should only contain things that at least two quickstarters use I'd say. So if there are files that are ONLY used by rshiny, the files should be in rshiny.

Comment (gerardcl) [reply common/scripts/create-rshiny.sh]: then we keep that in common, since that is used for jupyter too, but will rename it

Comment (michaelsauter) [reply common/scripts/create-rshiny.sh]: Haha. Cool. My recent PR #49 introduces some merge conflicts now - should be easy to resolve though ...

**Pull request merged**